substrate/frame/referenda/src/lib.rs

```diff
@@ -145,7 +144,7 @@ const ASSEMBLY_ID: LockIdentifier = *b"assembly";
 pub mod pallet {
 	use super::*;
 	use frame_support::{pallet_prelude::*, traits::EnsureOriginWithArg};
-	use frame_system::pallet_prelude::*;
+	use frame_system::pallet_prelude::{OriginFor, BlockNumberFor as SystemBlockNumberFor, ensure_root, ensure_signed_or_root, ensure_signed};
```

Comment:
Note that this could be replaced with:
```rust
	use super::{*, BlockNumberFor};
	use frame_support::{pallet_prelude::*, traits::EnsureOriginWithArg};
	use frame_system::pallet_prelude::{*, BlockNumberFor as SystemBlockNumberFor};
```

But as you prefer.